File: playground/java-practice/java/src/practice/factory/BurgerFactory.java
package practice.factory;

public class BurgerFactory {

    public static Burger createCheeseBurger() {
        Burger b = new Burger();

        b.addIngredient("Bun");
        b.addIngredient("Patty");
        b.addIngredient("Cheese");
        b.addIngredient("Bun");
        return b;
    }
}

File: playground/python-practice/designpatterns/factory/burgerfactory.py
from burger import Burger


class BurgerFactory:

    @staticmethod
    def create_cheese_burger():
        b = Burger()
        b.add_ingredient("Bun")
        b.add_ingredient("Patty")
        b.add_ingredient("Cheese")
        b.add_ingredient("Bun")
        return b
